dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -263,4 +263,15 @@ export class TfmFacilitiesRepo {
       'facilitySnapshot.ukefFacilityId': { $eq: ukefFacilityId },
     });
   }
+
+  /**
+   * Checks whether or not a document exists which has the matching UKEF facility ID
+   * @param ukefFacilityId - The UKEF facility ID
+   * @returns Whether a not a document with that id exists
+   */
+  public static async ukefFacilityIdExists(ukefFacilityId: string): Promise<boolean> {
```

Comment:
can this name include document?

Reply:
I'm not sure where you would include that in the name. In order to call this, you have to write  `TfmFacilitiesRepo.ukefFacilityIdExists` which I think implies that you are looking for a "tfm-facilities document"

Reply:
I think we should just tweak/simplify the documentation instead